src/commands/config/starboard/list.ts

```diff
@@ -33,7 +33,7 @@ const command: Command = {
         const embed = composeEmbedMessage(msg, {
           title: "Starboard Configuration",
           withoutFooter: true,
-          thumbnail: msg.guild.iconURL(),
+          thumbnail: msg.guild?.iconURL(),
```

Comment:
```suggestion
          thumbnail: msg.guild.iconURL(),
```
sau khi add logic check thì chỗ này k cần `?` nữa